Symfony/CS/Tests/IntegrationTest.php

```diff
@@ -33,6 +33,6 @@ protected static function getFixturesDir()
      */
     protected static function getTempFile()
     {
-        return self::getFixturesDir().'/tmp/tmp.php';
+        return static::getFixturesDir().'/tmp/tmp.php';
```

Comment:
oh, ehmm no :)

I meant, it should call its own implementation of `getFixturesDir` and not the parent one (`AbstractIntegrationTestCase`).